docs/site/BelongsTo-relation.md

```diff
@@ -102,34 +120,65 @@ export interface OrderRelations {
 export type OrderWithRelations = Order & OrderRelations;
 ```
 
-The standard naming convention for the foreign key property in the source model
-is `relation name` + `Id` (for example, Order.customerId).
+If the foreign key property name in the source model has to be customized
+(`customer_id` instead of `customerId` for example), the relation name has to be
+explicitly specified in the `name` attribute of the relation definition.
+Otherwise the _default relation name_ generates by LB4 (`customer_id` in this
+case) will be the same as the customized foreign key name, which is invalid.
 
-- If the foreign key property name in the source model has to be customized, the
-  relation name has to be explicitly specified in the `name` attribute of the
-  relation definition. In addition, if you have a corresponding `hasMany` or
-  `hasOne` relation in the target model (for example, a Customer has many
-  Orders), the `keyTo` attribute of that corresponding relation needs to be
-  stated explicitly. Check the relation metadata in
-  [hasMany](https://loopback.io/doc/en/lb4/HasMany-relation.html#relation-metadata)
-  and
-  [hasOne](https://loopback.io/doc/en/lb4/hasOne-relation.html#relation-metadata)
-  for more details.
-- In the following example, the foreign key property name is customized as
-  `cust_Id` instead of `customerId`. so the relation definition in the second
-  argument is explicitly passed to the `belongsTo` decorator.
+{% include warning.html content="Make sure that you have different names for the foreign key and the relation name in BelongsTo relations."%}
 
 ```ts
-class Order extends Entity {
+// import statements
+@model()
+export class Order extends Entity {
+  @property({
+    type: 'number',
+    id: true,
+  })
+  id: number;
+
+  @belongsTo(() => Customer, {name: 'customer'})
```

Comment:
Maybe put a comment here saying this is the correct way since from above you're warning what not to do